microsoft/PowerToys
# Add small delay to the search query before displaying results

## Summary of the Pull Request

**What is this about:**
This should address the performance issues mentioned in #11176. I've added code that will wait until a small delay occurs in user input before performing a search. This should prevent requerying on every character being typed.

**What is included in the PR:** 
I've implemented this using Rx, which required adding a reference to the System.Reactive nuget package to the PowerLauncher project.

**How does someone test / validate:** 
Perform a search query. Instead of search results being immediately displayed, the results should not be displayed until there is a 250ms delay in typing.

## Quality Checklist

- [X] **Linked issue:** #11176 

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
